mockredis/client.py

```diff
@@ -1444,6 +1444,8 @@ def _translate_range(self, len_, start, end):
         """
         Translate range to valid bounds.
         """
+        start = int(start)
```

Comment:
Added tester for client.py translation functions.
See comment about general test failure below.